src/assets/stylesheets/_social_cards.scss

```diff
@@ -221,3 +230,31 @@ $blueDarken: #0085ac;
     min-height: 157px;
   }
 }
+
+
+// --------------------------------------------------------------- //
+// scrivito_dark adaptions
+// --------------------------------------------------------------- //
+
+.scrivito_dark {
+  background: #3f3f3f;
+
+  .scrivito_detail_label span {
+    color: #eee;
+    text-shadow: 0 1px 1px rgba(0, 0, 0, 0.3), 0 0 4px rgba(0, 0, 0, 0.2);
+  }
+  .scrivito_detail_label span.headline { color: #aaa; }
+  .scrivito_notice_body { color: #aaa !important; }
+
+
+  .seo_card_input .input {
+    background: rgba(222, 222, 222, 0.2);
+    color: #eee;
+  }
+
+  [data-scrivito-editors-placeholder]:empty:not(:focus):before { color: #aaa!important;}
+
+  .seo_card_preview .card { background: rgba(222, 222, 222, 0.2); }
+  .seo_card_preview .card_text h5 { color: #ccc!important;}
+  .seo_card_preview .card_text p { color:#ddd!important;}
```

Comment:
* [ ] The creator preview is dark blue on dark grey

![image](https://user-images.githubusercontent.com/986170/140192471-58d16b24-b1ca-43a0-b5ab-cd0d53d24cdf.png)

Reply:
See https://github.com/Scrivito/scrivito_example_app_js/pull/428#discussion_r742639618